### PR TITLE
feat: add Technical SEO Analyst job role

### PR DESCRIPTION
**Pull request by marneysof**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

**Comment by BlueAcidFrog** [comment]
@aeneasr, would mind taking a close look at this job posting for an SEO Analyst, please?

File: src/markdown/jobs/technicalSEOanalyst.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+- **Position Type:** Full Time
+- **Location**: Munich, Germany
+- **Components**: DeepCrawl, Screaming Frog, and SEO Rada
+
+### What You'll Do
+
+- Lead technical SEO efforts in collaboration with web development and product teams
+- Assist on general SEO projects such as keyword research, content planning, and other internal SEO lead work
+- Monitor the Ory Corp website – finding and resolving technical or content issues that arise
```
Comment (aeneasr) [inline src/markdown/jobs/technicalSEOanalyst.mdx:38]:
```suggestion
- Monitor Ory‘s web presence – finding and resolving technical or content issues that arise
```

File: src/markdown/jobs/technicalSEOanalyst.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- Devise a best practice documentation scheme to communicate easi with engineering teams
+- Work with growth marketing people to foresee impact to SEO during A/B testing experiments. 
+- Communicate SEO roadmaps to relevant members of company leadership
+- Stay in the know on industry best practices and innovations in technical SEO
+- Assist on wider SEO objectives such as site migrations, page deprecations, and more
+
+
+### We’re looking for someone who has
+
+- 4 + years of experience working in technical SEO with multiple CMS platforms.
```
Comment (aeneasr) [inline src/markdown/jobs/technicalSEOanalyst.mdx:50]: @BlueAcidFrog @marneysof is this a strict requirement? or can we soften it up?

Comment (BlueAcidFrog) [reply src/markdown/jobs/technicalSEOanalyst.mdx:50]: @aeneasr we can but I wouldn't recommend it. If needed we can go for 3+ years.

Comment (aeneasr) [reply src/markdown/jobs/technicalSEOanalyst.mdx:50]: ok then keep as is

**Pull request merged**